packages/e2e-selenium/.env.template

```diff
@@ -0,0 +1,2 @@
+LOCAL_BROWSERSTACK_USER=
+LOCAL_BROWSERSTACK_KEY=
```

Comment:
Do you have an [EditorConfig plugin](https://editorconfig.org/) installed for your editor?

There's a project [`.editorconfig` configuration](https://github.com/WordPress/gutenberg/blob/master/.editorconfig) which will enforce final newlines for those who have a plugin installed. I'd worry this file could be prone to unexpected changes. Ideally we normalize toward this configuration as the baseline target.

(Same goes for a couple other files in this branch)

Reply:
Fixed them all.